jazz_test-lambda/README.md

```diff
@@ -0,0 +1,27 @@
+## API for Testing Lambda Functions.
+Provides an API for Testing a lambda function and returns the Execution Status
+#### Request format in JSON
+```
```

Comment:
Please update this as per the new swagger contract (see my comments).

Reply:
Awaiting clarification

Reply:
done